bundle-size/.env.example

```diff
@@ -4,6 +4,9 @@ WEBHOOK_SECRET=0123456789abcdef0123456789abcdef01234567
 WEBHOOK_PROXY_URL=https://smee.io/0123456789abcdef  # For local development only
 PRIVATE_KEY=0123456789abcdefghijklmnopqrstuvwxyz  # Base64 encoded private key from GitHub
 TRAVIS_IP_ADDRESSES=1.1.1.1,8.8.8.8  # Remove this line for local development, update based on https://docs.travis-ci.com/user/ip-addresses/ otherwise
+APPROVER_TEAMS=123,234,345  # List of GitHub team IDs whose members can approve bundle-size changes
+REVIEWER_TEAMS=123  # List of GitHub team IDs from which a random member will be assigned as a reviewer. SHOULD be a subset of APPROVER_TEAMS (this is not verified in runtime)
+REVIEWER_TEAM_NAMES=@ampproject/wg-runtime  # As-is text to add to the message on who can be tagged to get a reviewer
```

Comment:
I assume this can this contain more than one team?

Reply:
Yes, this is just replaced as text